dwave/system/temperatures.py

```diff
@@ -109,137 +145,243 @@ def effective_field(bqm,
-       >>> E = effective_field(bqm,samples,current_state_energy=True)
-       >>> print('Cost to flip spin against current assignment', E)     
+       >>> E = effective_field(bqm, samples, current_state_energy=True)
+       >>> print('Cost to flip spin against current assignment', E)
        Cost to flip spin against current assignment (array([[-1., -2., -2., -2., -1.]]), [0, 1, 2, 3, 4])
 
-    '''
+    """
     if samples is None:
-        samples = np.ones(shape=(1,bqm.num_variables))
+        samples = np.ones(shape=(1, bqm.num_variables))
         labels = bqm.variables
     else:
         samples, labels = dimod.sampleset.as_samples(samples)
 
     if bqm.vartype is dimod.BINARY:
-        bqm = bqm.change_vartype('SPIN', inplace=False)
-        samples = 2*samples - 1
+        bqm = bqm.change_vartype(dimod.SPIN, inplace=False)
+        samples = 2 * samples - 1
 
-    h, (irow, icol, qdata), offset = bqm.to_numpy_vectors(
-        variable_order=labels)
+    h, (irow, icol, qdata), offset = bqm.to_numpy_vectors(variable_order=labels)
     # eff_field = h + J*s OR diag(Q) + (Q-diag(Q))*b
-    effective_fields = np.tile(h[np.newaxis,:],(samples.shape[0],1))
+    effective_fields = np.tile(h[np.newaxis, :], (samples.shape[0], 1))
     for sI in range(samples.shape[0]):
-        np.add.at(effective_fields[sI,:],irow,qdata*samples[sI,icol])
-        np.add.at(effective_fields[sI,:],icol,qdata*samples[sI,irow])
+        np.add.at(effective_fields[sI, :], irow, qdata * samples[sI, icol])
+        np.add.at(effective_fields[sI, :], icol, qdata * samples[sI, irow])
 
     if current_state_energy is True:
-        #Ising: eff_field = 2*s*(h + J*s)
-        effective_fields = 2*samples*effective_fields
-
-    return (effective_fields,labels)
-
-def maximum_pseudolikelihood_temperature(bqm=None,
-                                         sampleset=None,
-                                         site_energy=None,
-                                         num_bootstrap_samples=0,
-                                         seed=None,
-                                         T_guess=None,
-                                         optimize_method='bisect',
-                                         T_bracket=(1e-3,1000)) -> Tuple[float,np.ndarray]:
-    r'''Returns a sampling-based temperature estimate.
-
-    The temperature T parameterizes the Boltzmann distribution as 
-    :math:`P(x) = \exp(-H(x)/T)/Z(T)`, where :math:`P(x)` is a probability over a state space, 
-    :math:`H(x)` is the energy function (BQM) and :math:`Z(T)` is a normalization. 
-    Given a sample set (:math:`S`), a temperature estimate establishes the 
+        effective_fields = 2 * samples * effective_fields
+
+    return (effective_fields, labels)
+
+
+def background_susceptibility_Ising(
+    h: Union[np.ndarray, dict], J: Union[np.ndarray, dict]
+) -> Tuple:
+    """Create the field and Hamiltonian for the `background susceptibility correction`_
```

Comment:
Linking doesn't work.
```suggestion
    """Create the field and Hamiltonian for the background susceptibility correction.
```